lib/generator.js

```diff
@@ -751,8 +751,11 @@ class Generator {
    * @private
    */
   async launchHook(hookName, hookArguments) {
-    const disabledHooks = this.disabledHooks[hookName] || [];
+    let disabledHooks = this.disabledHooks[hookName] || [];
     if (disabledHooks === true) return;
+    if (typeof disabledHooks === "string") {
+      disabledHooks = [disabledHooks];
+    }
```

Comment:
I know it is just 3 lines of code, you could even do 1 with `if (typeof disabledHooks === "string") disabledHooks = [disabledHooks];`


but did you consider that `if (disabledHooks.includes(hook.name)) return;` could work without the above code because string also has `includes` function. Or am I overlooking something?

Reply:
Yes, string has also `includes` function, but includes for string is something like a wrapper for regexp so `disabledHooks = [disabledHooks];` is easier way to disable hook -> e.g. if disabledHooks will have `foobar` name  and then single hook function will have `foo` name, then condition `if (disabledHooks.includes(hook.name))` will return true

Reply:
@magicmatatjahu did you rebase? strange that this comment got outdated? please don't rebase

Anyway, valid point, I agree with you

Reply:
This is outdated because, as you suggested, change 3 lines to 1 line :P